jscomp/others/belt_List.ml

```diff
@@ -692,6 +692,13 @@ let setAssocU xs x k eq =
 
 let setAssoc xs x k eq = setAssocU xs x k (fun [@bs] a b -> eq a b)
 
+let sort xs cmp =
+  let arr = ref (toArray xs) in
+  begin
+    Belt_SortArray.stableSortInPlaceBy !arr cmp;
```

Comment:
A.stableSortInPlaceBy -- it is already aliased in the beginning

Reply:
Belt_Array is aliased as A in this module. It doesn't have the sort functions from Belt_SortArray. Should Belt_SortArray be opened in Belt_Array?

Reply:
This is fine then, sorry my mistake